internal/msg/errormsg.go

```diff
@@ -0,0 +1,155 @@
+package msg
+
+// appstore uploading
+const (
+	// FailedToUpload indicates the upload failure
+	FailedToUpload = "failed to upload project"
+	// FileNotFound indicates file not found
+	FileNotFound = "%s: file not found"
+	// UploadingTimeout is a the message to warn the user that its upload reach the timeout.
+	UploadingTimeout = `Failed to upload the project because it took too long. `
+)
+
+// cmd setting
+const (
+	// InvalidUsername indicates invalid username
+	InvaildUsername = "invalid username"
+	// EmptyUsername asks user to type a username
+	EmptyUsername = "you need to type a username"
+	// InvalidAccessKey indicates invalid key
+	InvalidAccessKey = "invalid access key"
+	// EmptyAccessKey asks user to type an access key
+	EmptyAccessKey = "you need to type an access key"
+	// EmptyCredentials indicates no credentials
+	EmptyCredentials = "no credentials available"
+	// InvalidSelectedFramework indicates invalid framework
+	InvalidSelectedFramework = "invalid framework selected"
+	// InvalidCredentials indicates invalid credentials
+	InvalidCredentials = "invalid credentials provided"
+	// UnableToCheckCredentials
+	UnableToCheckCredentials = "unable to check credentials"
+	// MissingFrameworkVersion indicates empty framework version
+	MissingFrameworkVersion = "no %s version specified"
+	// MissingCypressConfig indicates no cypress config file
+	MissingCypressConfig = "no cypress config file specified"
+	// MissingPlatformName indicates no platform name
+	MissingPlatformName = "no platform name specified"
+	// MissingBrowserName indicates no browser name
+	MissingBrowserName = "no browser name specified"
+	// MissingApp indicates no app
+	MissingApp = "no app provided"
+	// MissingTestApp indicates no testApp
+	MissingTestApp = "no testApp provided"
+	// MissingDeviceOrEmulator indicates no device or emulator
+	MissingDeviceOrEmulator = "either device or emulator configuration needs to be provided"
+	// MissingDevice indicates no device provided
+	MissingDevice = "no device provided"
+	// EmptySuiteName is thrown when a flag is specified that has a dependency on the --name flag.
+	EmptySuiteName = "adhoc suite parameters can only be used with a new adhoc suite by setting --name"
```

Comment:
Should'nt the name mention that this is only for adhoc / cli-driven approach ?

Reply:
make sense!